src/System.Net.HttpListener/tests/HttpListenerRequestTests.cs

```diff
@@ -186,7 +186,6 @@ public static IEnumerable<object[]> ContentEncoding_TestData()
             });
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
```

Comment:
Wow.  Good catch.  Yes, this needs a Theory attribute. We'll need to submit another PR to fix that.